tutorials/metrics/metrics_file_regex_with_linenum.yml

```diff
@@ -0,0 +1,27 @@
+buildspecs:
+  metric_file_regex:
+    executor: generic.local.bash
+    type: script
+    description: capture result metric from file path
+    run: echo -e "HPCG result is INVALID with a GFLOP/s rating of=28.1215\n
+                  HPCG result is VALID with a GFLOP/s rating of=68.9888" > hpcg.txt
+    tags: tutorials
+    metrics:
+      second_line:
+        type: float
+        file_regex:
+          exp: '(\d+\.\d+)$'
+          linenum: 1
```

Comment:
i wonder if we did `-1` it would be the same as `1` in this example since content of file is only two lines. This could be good way perhaps changing the text `second_line` to `last_line`


0:  first line of file
1:  second line
-1:  last line of file (2nd line)
-2: 2nd last line of file (first line)

Reply:
It would also allow us to explain that negative index also work and you have a duplicate example with `regex` with `linenum` so this might complement well.

Reply:
👍